interfaces/core.go

```diff
@@ -135,6 +128,13 @@ func (conn *Connection) Interface() string {
 	return conn.plugInfo.Interface
 }
 
+func NewConnRef(plug *snap.PlugInfo, slot *snap.SlotInfo) *ConnRef {
```

Comment:
Can you please document this?

`// NewConnRef returns a new ConnRef for the given plug and slot`